posydon/config.py

```diff
@@ -3,6 +3,32 @@
 
 load_dotenv()
 
+def ensure_path(name):
+    """Get the PATH set in the environment
```

Comment:
```suggestion
    """Get the path name set in the environment
```
I would like to suggest not using PATH in the doc string, since it's an environment variable in Linux and Mac (maybe Windows). It's confused me.
Same for the PR name ;)